src/pudl/output/ferc1.py

```diff
@@ -2496,11 +2496,16 @@ def forest_leaves(self: Self) -> list[NodeId]:
     def orphans(self: Self) -> list[NodeId]:
         """Identify all nodes that appear in metadata but not in the full digraph."""
         nodes = self.full_digraph.nodes
-        return [
-            NodeId(*n)
-            for n in self.exploded_meta.set_index(self.calc_cols).index
-            if n not in nodes
-        ]
+        orphans = []
+        for idx_cols in [self.calc_cols, self.parent_cols]:
+            orphans.extend(
+                [
+                    NodeId(*n)
+                    for n in self.exploded_calcs.set_index(idx_cols).index
+                    if n not in nodes
+                ]
+            )
+        return list(set(orphans))
```

Comment:
this is the only other place that the `explode_meta` was being used. I think it actually makes more sense to use the `exploded_calcs` here to determine the og set of nodes here because that's what the forest was actually build on and the metadata could be quite different if we weren't being careful in the prep